packages/www/hooks/use-analyzer.tsx

```diff
@@ -1,6 +1,8 @@
-import { useContext, createContext, ReactNode } from "react";
+import EventSource from "eventsource";
```

Comment:
Oh neat. So is this a totally custom fetch-based `eventsource` implementation now? I can't imagine that's a problem but cool.